src/components/Modal/BaseModal.js

```diff
@@ -144,7 +144,7 @@ class BaseModal extends PureComponent {
 
                         const modalPaddingStyles = StyleUtils.getModalPaddingStyles({
                             safeAreaPaddingTop,
-                            safeAreaPaddingBottom,
+                            safeAreaPaddingBottom: insets.bottom === 0 ? 12 : safeAreaPaddingBottom,
```

Comment:
Should this logic live in `StyleUtils.getModalPaddingStyles` instead? If we pass the insets, we can calculate this automatically there rather than patching this change on to BaseModal